docker/dockerfiles/toolchains/toolchains-ubuntu.Dockerfile

```diff
@@ -7,7 +7,7 @@ FROM ubuntu:22.04
 RUN apt-get update -y && \
     apt-get install --no-install-recommends -y \
     automake binutils-dev bison byacc ccache flex libiberty-dev libtool maven zip python3 python-is-python3 make cmake gcc g++ default-jdk git patch lld bzip2 \
-    wget unzip curl vim tree net-tools openssh-client xz-utils && \
+    wget unzip curl vim tree net-tools openssh-client xz-utils gh && \
```

Comment:
this makes the gh only works for ubuntu platform, not working for centos7. possibly worth comment that it won't work for centos7 at all.

Reply:
Added in centos7 docker file to install gh. Tested locally.